src/Domain/Service/MessagePresenter.php

```diff
@@ -42,11 +42,11 @@ public function strip(string $message): string
 
     private function convertNewLinesToParagraphs(string $content): string
     {
-        $content = str_replace("\r\n", "\n", $content);
-        $content = str_replace("\r", "\n", $content);
-        $pattern = '/^(.*)$/m';
-
-        return preg_replace($pattern, '<p>$1</p>', $content);
+        return preg_replace(
```

Comment:
Previous version suggest they were previously no `<br/>` in the `$content` string, is that no longer the case ?
Might be good to have a few tests to mock what's expected in input and output.

Reply:
About this enhancement, the problem and what's expected was not clearly written. My bad. Thus I created a special card about it : https://trello.com/c/Fx5GViCu/539-un-retour-%C3%A0-la-ligne-devient-un-saut-de-ligne-sur-la-page-profil 
Hope it will help.

Reply:
We were replacing new lines with new paragraphs, this PR changes to replace them by line breaks

Reply:
Actually, the tests existed, but I forgot to update them :sweat_smile: 

Done !